docker-compose.yml

```diff
@@ -324,7 +324,7 @@ services:
     bridge_redis:
         command: [redis-server, --appendonly, 'yes']
         hostname: redis
```

Comment:
@isentropy If the hostname is redis then does it mean that bridge_redis has been unused? Do you know whats the hostname for Streamr redis instance?

Reply:
@isentropy hostname is now `bridge-redis` in master. I suppose that the Streamr redis hostname is `redis`.